### DISCUSSION TITLE
How can I customize the next/previous tab shortcut?

### DISCUSSION THREAD

**Discussion by rfpozo** [sumatrapdfreader/sumatrapdf#2967] [Q&A]
In https://github.com/sumatrapdfreader/sumatrapdf/blob/master/src/Commands.h I found the commands "CmdNextTab" and "CmdPrevTab" but I can't use them to define the shortcuts on the SumatraPDF-settings.txt file. I would like to define J and K for moving between tabs. Thanks!

Shortcuts [
[
Cmd = CmdNextTab
Key = T
]
[
Cmd = CmdPrevTab
Key = t
]
]

**Comment by GitHubRulesOK** [comment]
***Works for me*** ignore the layout I can go t=left T=right perhaps check version is 3.4.5 ish
![image](https://user-images.githubusercontent.com/12754913/191108355-82e46da4-f91a-4cf7-979a-850ab4b4eba1.png)

**Reply by fschae** [reply]
Does not work for me either, version is newest. Also the [command reference](https://www.sumatrapdfreader.org/docs/Commands) does not have them listed. Trying to map them to `h` and `l`. Can you verify that is also works for you with the current version? Or could you imagine, that other options influence it aswell?
![IMG_000853](https://user-images.githubusercontent.com/91015021/227710170-b29bf85f-869e-4441-9d32-ff3986ea3484.png)

**Comment by kjk** [comment]
Those are only available in pre-release version https://github.com/sumatrapdfreader/sumatrapdf/discussions/2735
